merlin/dataloader/ops/embeddings.py

```diff
@@ -98,10 +98,23 @@ def compute_output_schema(
             col_schemas.append(col_schema)
         id_schema = input_schema.column_schemas[self.lookup_key]
         embedding_dim = self.embeddings.shape[1]
+
+        new_tags = [Tags.EMBEDDING]
+        propagated_tags = [
+            Tags.LIST,
+            Tags.SEQUENCE,
+            Tags.USER,
+            Tags.ITEM,
+            Tags.CONTEXT,
+            Tags.SESSION,
+        ]
+        for tag in propagated_tags:
```

Comment:
Is the idea that this is going to be updated with an exclusion of tags based on the `TAG_COLLISIONS[Tags.EMBEDDING]` in place of the allow-list propgation here. That way it will automaticaly propage any valid tags? including custom tags added by a user to their dataset (since tags can also be strings)

Reply:
At first we thought so, but we then realized that it wasn't safe to assume that any tag should automatically be propagated unless we knew specifically that the embedding op wouldn't change the applicability of the tag, so we switched tactics from a deny-list to an allow-list. The deny-list changes in Merlin Core are still important though, even though they're not really used here, because you can manually apply the EMBEDDING tag, which ought to remove any other tags that are not compatible with EMBEDDING.